examples/esp32spi_wpa2ent_aio_post.py

```diff
@@ -0,0 +1,57 @@
+import time
+import board
+import busio
+from digitalio import DigitalInOut
+import neopixel
+from adafruit_esp32spi import adafruit_esp32spi
+from adafruit_esp32spi.adafruit_esp32spi_wifimanager import ESPSPI_WiFiManager
+
+print("ESP32 SPI WPA2 Enterprise webclient test")
+
+# Get wifi details and more from a secrets.py file
+try:
+    from secrets import secrets
+except ImportError:
+    print("WiFi secrets are kept in secrets.py, please add them there!")
+    raise
+
+# If you are using a board with pre-defined ESP32 Pins:
```

Comment:
Check out how we automatically switch between the ESP32 pin definitions in CircuitPython_IO examples: https://github.com/adafruit/Adafruit_CircuitPython_AdafruitIO/blob/master/examples/adafruit_io_simpletest_data.py#L25